temba/msgs/views.py

```diff
@@ -43,6 +43,7 @@ def send_message_auto_complete_processor(request):
         completions.insert(4, dict(name='contact.tel_e164', display=unicode(_("Contact Phone - E164"))))
         completions.insert(5, dict(name='contact.groups', display=unicode(_("Contact Groups"))))
         completions.insert(6, dict(name='contact.uuid', display=unicode(_("Contact UUID"))))
+        completions.insert(6, dict(name='contact.language', display=unicode(_("Contact Language"))))
```

Comment:
I think I'm missing something in terms of alphabetizing this, unless this test was already failing. You've got a strange and mysterious alphabet here if ``tel_e164``` or "Contact Phone" comes before either ```contact.groups``` or "Contact Groups"

Here, though, i botched the insertion and didn't increment the position